docs/handling_security_vulnerability_disclosures.md

```diff
@@ -4,8 +4,21 @@ The below process describes the actions that the rust-vmm maintainers need to
 take when they are disclosed on a security vulnerability via
 [the security disclosure process](https://github.com/rust-vmm/community/blob/master/SECURITY.md).
 
-If during the process it becomes apparent there is no security impact,
+- If during the process it becomes apparent there is no security impact,
 shift to treating as a public issue.
+- When updating the changelog in the private fork under a security advisory use the advisory ID to 
+  prefix the change description in the changelog e.g.
+  ```
+  ## v0.12.1
+
+  ### Changed
+
+  - [[GHSA-875g-mfp6-g7f9](https://github.com/rust-vmm/vmm-sys-util/security/advisories/GHSA-875g-mfp6-g7f9)]
+    Fixed a bug in serde implementation of FamStructWrapper which allowed out of bounds memory access from Rust-safe code.
```

Comment:
Can we break this line into two? GitHub is giving me an ugly horizontal scrollbar :(

Reply:
I've split the text into 2 lines, but I cannot split the link as this would not work in markdown.

Reply:
ah, sorry, I only meant the text, thanks!